File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int res=0;
        for(String i:words){
            int flag=0;
            for(int j=0;j<i.length();j++){
                if(!allowed.contains(String.valueOf(i.charAt(j)))){
                    flag=1;
                }
                if(flag==1)
                    break;
            }
            if(flag==0){
                res++;
            }
        }
        return res;
        
    }
}

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        res=0
        for i in words:
            flag=0
            for j in range(len(i)):
                if i[j] not in allowed:
                    flag=1
                if flag==1:
                    break
            if flag==0:
                res+=1
        return res

        